languages/README.md

```diff
@@ -0,0 +1,106 @@
+# Instructions for CloudEvents Specification in Multi-languages
+
+<!-- no verify-specs -->
+
+## Abstract
+
+This page contains information about how to translate a document from scratch, definition of roles and responsibilities 
+as well as guidelines that translators and maintainers should follow.
+
+## Status of this document
+
+This is a working draft. Changes could be made anytime.
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Roles](#roles)
+- [How to translate](#how-to-translate)
+- [Translation style guide](#translation-style-guide)
+
+## Overview
+
+We welcome you to join us and contribute to the CloudEvents spec!
+
+CloudEvents is a specification for describing event data in common formats to provide interoperability across services, platforms and systems.
+An increasing number of developers are becoming interested in CloudEvents.
+We encourage you to translate the spec into a specific language since the language barrier might hinder the spread of CloudEvents.
+
+This page contains information about how to translate a document from scratch, definition of roles and responsibilities
+as well as guidelines that translators and maintainers should follow.
+
+## Roles
+
+There are two categories of roles:
+
+1. **Translator.** This is anyone who participates in the translation of any
+   CloudEvents documents. One becomes a formal translator once a Pull Request(PR) of his/her translation is merged.
+
+2. **Maintainer.** A maintainer is responsible for checking new translation, 
```

Comment:
I'm not sure we need formal roles since a "Maintainer" role doesn't really exist in the sense that they can't merge PRs. Is there any reason we can't just treat these PRs like any other PR that the group processes?

Reply:
Perhaps what you really mean to call these are "reviewers" or "approvers"? We can certainly make it so the group doesn't merge a PR until these folks give their LGTM

Reply:
Yeah, the role I thought is more like to be  a "language reviewer" who should check the translation and leave a LGTM before group merge that PR. I'll change it now.